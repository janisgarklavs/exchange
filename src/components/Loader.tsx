import React from "react";

import "./Loader.css";

export const Loader: React.SFC = () => (
  <div className="spinner">
    <div className="bounce1" />
    <div className="bounce2" />
    <div className="bounce3" />
  </div>
);
