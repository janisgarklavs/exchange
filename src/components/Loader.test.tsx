import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import { Loader } from "./Loader";

describe("<Loader />", () => {
  test("it renders correctly", () => {
    const component = shallow(<Loader />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
