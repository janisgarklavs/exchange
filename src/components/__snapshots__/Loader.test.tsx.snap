// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> it renders correctly 1`] = `
<div
  className="spinner"
>
  <div
    className="bounce1"
  />
  <div
    className="bounce2"
  />
  <div
    className="bounce3"
  />
</div>
`;
